ssp_author_demo/test_system/ac/ac-14.md

```diff
@@ -1,24 +1,37 @@
-# ac-14 - Access Control Permitted Actions Without Identification or Authentication
+---
+sort-id: ac-14
+x-trestle-sections:
+  guidance: Control Guidance
+---
 
-## Control Description
+# ac-14 - \[Access Control\] Permitted Actions Without Identification or Authentication
+
+## Control Statement
+
+- \[a.\] Identify organization-defined user actions that can be performed on the system without identification or authentication consistent with organizational mission and business functions; and
 
-- \[a.\] Identify \[ac-14_prm_1 = organization-defined user actions\] that can be performed on the system without identification or authentication consistent with organizational mission and business functions; and
 - \[b.\] Document and provide supporting rationale in the security plan for the system, user actions not requiring identification or authentication.
 
+## Control Control Guidance
```

Comment:
That is a bit odd, why does it repeats?

Reply:
Good catch.  It isn't a bug but it means the section name should simply be "Guidance" and not "Control Guidance" because Control is put in there automatically.  I will change it.